yamls/NWA12/CEFI_NWA12_cobalt.yaml

```diff
@@ -0,0 +1,88 @@
+# POSTPROCESSING GUIDE
+#
+# The following yaml is designed to postprocess the ocean_monthly component of the 2024_06 version of the CEFI_NWA12_COBALT_V1 experiment
+# if postprocessing data from a different version of this experiment, be sure to change the FRE_STEM variable before postprocessing
+#
+# Quick start instructions and source code can be found here: https://github.com/NOAA-GFDL/fre-cli/tree/main/fre/pp
+#
+# 0.) Load module on GFDL Analysis: module load fre/canopy
+#
+# For modularity and compatibility with with cylc workflow engine, fre/canopy splits up the post processcing process into several sub steps:
+#
+# 1.) Checkout the git repo containing postprocessing scripts and related files with the folowing command:
+# fre pp checkout -e CEFI_NWA12_COBALT_V1 -p gfdl.ncrc5-intel22 -t prod
+#
+# 2.) Combine your main yaml and experiment yamls into a single yaml, then set up the cylc-src dir with the configure-yaml command:
+# fre yamltools combine-yamls -e CEFI_NWA12_COBALT_V1 -p gfdl.ncrc5-intel22 -t prod -y CEFI_NWA12_cobalt.yaml
+# fre pp configure-yaml -e CEFI_NWA12_COBALT_V1 -p gfdl.ncrc5-intel22 -t prod -y combined-CEFI_NWA12_COBALT_V1.yaml
+#
+#     2a.) Fre/canopy does not automatically create the pp dir for you, so you will have to mkdir this first to pass the validator:
+#     mkdir /archive/$USER/fre/cefi/NWA/2024_06/gfldl.ncrc5-intel22-prod/pp
```

Comment:
Normally the archive path should contain experiment name such as:
`/archive/$USER/fre/cefi/NWA/2024_06/CEFI_NWA12_COBALT_V1/gfdl.ncrc5-intel22-prod/pp`